meta-mender-intel/scripts/manifest-intel.xml

```diff
@@ -1,12 +1,13 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-  <manifest>
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote fetch="git://git.yoctoproject.org" name="yocto"/>
+  <remote fetch="git://git.openembedded.org" name="oe"/>
 
-  <remote fetch="git://git.yoctoproject.org"        name="yocto"/>
+  <default revision="kirkstone" sync-j="4"/>
```

Comment:
I think this will clash with the upper level `scripts/mender.xml`, repo always gives me an error if there are two `<default/> statements.

Reply:
How do you use the top level `scripts/mender.xml` ?

Following the instructions and documentation, it doesn't trigger any issues:
```
    $ mkdir mender-<vendor/soc name>
    $ cd mender-<vendor/soc name>
    $ repo init \
           -u https://github.com/mendersoftware/meta-mender-community \
           -m meta-mender-<vendor/soc name>/scripts/manifest-<vendor/soc name>.xml \
           -b kirkstone
    $ repo sync
```
and in meta-mender-intel:
```
mkdir mender-intel && cd mender-intel
repo init -u https://github.com/mendersoftware/meta-mender-community \
           -m meta-mender-intel/scripts/manifest-intel.xml \
           -b kirkstone
repo sync
source setup-environment intel
bitbake core-image-base
```

In addition, if I grep in the repository, there's 6 other layers using the default.

IMO, either it's removed on all the others as well, or we can it repo's wide. Opinion?

Reply:
@TheYoctoJester what about the above?